dataland-document-manager/src/main/kotlin/org/dataland/documentmanager/controller/DocumentController.kt

```diff
@@ -82,6 +84,16 @@ class DocumentController(
         )
     }
 
+    override fun replaceDocumentMetaInfo(
```

Comment:
Is there a reason to not call the put endpoint 'putDocumentMetaInfo'?
```suggestion
    override fun putDocumentMetaInfo(
```